sentry/src/main/java/io/sentry/transport/HttpTransport.java

```diff
@@ -131,6 +146,8 @@ public HttpTransport(
     this.readTimeout = readTimeoutMillis;
     this.options = options;
     this.bypassSecurity = bypassSecurity;
+    this.sslSocketFactory = sslSocketFactory;
```

Comment:
Do you mind also deleting all this `bypassSecurity` code?
You approach is preferable and we'll shift over to it.

Reply:
okay, i will delete them.

regarding to my own testing 

this is how I initialize Sentry
![image](https://user-images.githubusercontent.com/9247289/94279178-f54de200-ff75-11ea-902d-fdb56f6c2b30.png)

MainPresenter sslcontext and hostnameverifier
![image](https://user-images.githubusercontent.com/9247289/94279630-95a40680-ff76-11ea-8a3b-63a451f6fb05.png)

![image](https://user-images.githubusercontent.com/9247289/94279716-b53b2f00-ff76-11ea-8273-cda67d474388.png)

envelope sent
![image](https://user-images.githubusercontent.com/9247289/94280106-38f51b80-ff77-11ea-9364-f76720caeb01.png)